.github/workflows/prune-attachments.yml

```diff
@@ -0,0 +1,37 @@
+name: Prune Old Allure Attachments
+on:
+  schedule:
+    - cron: '0 16 * * 0' # Weekly, 16:00 UTC Sunday
+
+jobs:
+  prune:
+    runs-on: self-hosted
```

Comment:
could be ubuntu-latest I guess

Reply:
seeing the sh commands run below, we definitely need it to be enforced to linux (which it isn't right now)